.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+    - 8
+install:
+    - npm i
+    - npm i -g codecov
+script:
+    - npm run check
+    - npm test
+after_success:
+    - codecov
```

Comment:
Missing newline at the end of file. Really, you should use some githook that will check this for you…

Reply:
Fixed